### PR TITLE
Add more conversions between Time/Duration and `std` types, and add `Time::now()`.

### PR DESCRIPTION
**Pull request by de-vri-es**
This PR implements conversion from `Duration` to `std::time::Duration`, conversions between `Time` and `std::time::SystemTime`, and adds `Time::now()` to get the current time.

It also adds unit tests for the new conversions.

### CODE REVIEW

File: ros_message/src/time.rs [hunk 1]
```diff
@@ -37,6 +38,28 @@ impl Time {
...
+    /// # #[derive(Default)]
+    /// # struct Message {
+    /// #   header: Header,
+    /// # }
+    /// # let mut message = Message::default();
+    /// message.header.stamp = Time::now();
+    /// ```
+    #[inline]
+    pub fn now() -> Self {
+        time::SystemTime::now().into()
```
Comment (romainreignier) [inline ros_message/src/time.rs:60]: In `roscpp`, `ros::Time::now()` might returns the simulated time so this implementation is not correct I think.
And we already have `rosrust::now()`.
But something like `rosrust::wall_time::now()` using `std::time::SystemTime::now()` would be nice.

Comment (de-vri-es) [reply ros_message/src/time.rs]: Implemented your suggestion :)

File: ros_message/src/time.rs [hunk 2]
```diff
@@ -306,8 +326,24 @@ impl ops::Neg for Duration {
 impl From<time::Duration> for Duration {
     fn from(std_duration: time::Duration) -> Self {
         Duration {
-            sec: std_duration.as_secs() as i32,
+            sec: std_duration.as_secs().try_into().unwrap(),
```
Comment (adnanademovic) [inline ros_message/src/time.rs:329]: `expect("Durations longer than 68 years are not supported by the ROS time format")`

File: ros_message/src/time.rs [hunk 3]
```diff
@@ -133,6 +134,25 @@ impl cmp::Ord for Time {
...
+            nsec: elapsed.subsec_nanos(),
+        }
+    }
+}
+
+impl From<Time> for time::SystemTime {
+    fn from(other: Time) -> Self {
+        let elapsed = time::Duration::new(other.sec.into(), other.nsec);
+        time::SystemTime::UNIX_EPOCH + elapsed
+
```
[the remark this answers is not in the document]
Comment (de-vri-es) [reply ros_message/src/time.rs:152]: Woops, good point :)

**Comment by de-vri-es** [comment]
Fixed the tests :)

**Pull request merged**